cumulus_library_opioid/vocab/static_builder.py

```diff
@@ -176,4 +194,4 @@ def prepare_queries(
                         remote_table_cols_types=table.parquet_types,
                     )
                 )
-                progress.advance(task)
+                progress.advance(task)                
```

Comment:
:scissors:

Reply:
what a good reminder to do a ruff pass, got too used to it being automated.